keras_cv/models/object_detection/retina_net/retina_net_test.py

```diff
@@ -124,28 +158,85 @@ def test_fit_coco_metrics(self):
             box_loss=keras_cv.losses.SmoothL1Loss(l1_cutoff=1.0, reduction="none"),
             reduction="sum",
         )
+
         retina_net.compile(
-            optimizer="adam",
+            optimizer=optimizers.SGD(learning_rate=0.25),
+            loss=loss,
+        )
+
+    # TODO(lukewood): configure for other coordinate systems.
+    @pytest.mark.skipif(
+        "INTEGRATION" not in os.environ,
+        reason="Takes a long time to run, only runs when INTEGRATION "
+        "environment variable is set.  To run the test please run: \n"
+        "`INTEGRATION=true pytest "
+        "keras_cv/models/object_detection/retina_net/retina_net_test.py -k "
+        "test_fit_coco_metrics -s`",
+    )
+    @parameterized.named_parameters(
+        ("xywh", "xywh"),
+    )
+    def test_fit_coco_metrics(self, bounding_box_format):
+        retina_net = keras_cv.models.RetinaNet(
+            classes=1,
+            bounding_box_format=bounding_box_format,
+            backbone="resnet50",
+            backbone_weights=None,
+            include_rescaling=False,
+        )
+        loss = keras_cv.losses.ObjectDetectionLoss(
+            classes=1,
+            classification_loss=keras_cv.losses.FocalLoss(
+                from_logits=True, reduction="none"
+            ),
+            box_loss=keras_cv.losses.SmoothL1Loss(l1_cutoff=1.0, reduction="none"),
+            reduction="sum",
+        )
+
+        retina_net.compile(
+            optimizer=optimizers.Adam(),
             loss=loss,
             metrics=[
                 keras_cv.metrics.COCOMeanAveragePrecision(
                     class_ids=range(1),
-                    bounding_box_format="xywh",
-                    name="Standard MaP",
-                )
+                    bounding_box_format=bounding_box_format,
+                    name="MaP",
+                ),
+                keras_cv.metrics.COCORecall(
+                    class_ids=range(1),
+                    bounding_box_format=bounding_box_format,
+                    name="Recall",
+                ),
             ],
         )
 
         xs, ys = _create_bounding_box_dataset(bounding_box_format)
-        retina_net.fit(x=xs, y=ys, epochs=1)
-        metrics = retina_net.evaluate(x=xs, y=ys, return_dict=True)
-        self.assertIn("Standard MaP", metrics)
+
+        for _ in range(50):
```

Comment:
This seems to be a too expansive to be a unit test, how about move this part to be a different test, and still keep the main part of this to be a unit test (for better coverage.)

Reply:
Sounds good, we can start an integration tests directory perhaps?

Reply:
Lets keep this here with the skip pattern for now, and create an integration test directory as a follow up PR.

https://github.com/keras-team/keras-cv/issues/700